projects/stretch_ovmm/configs/agent/floorplanner_eval.yaml

```diff
@@ -1,34 +1,33 @@
-BASE_TASK_CONFIG_PATH: projects/stretch_ovmm/configs/task/floorplanner_val.yaml
-
-NO_GPU: 1                 # 1: ignore IDs above and run on CPU, 0: run on GPUs with IDs above
-NUM_ENVIRONMENTS: 35      # number of environments (per agent process)
+NO_GPU: 0                 # 1: ignore IDs above and run on CPU, 0: run on GPUs with IDs above
+NUM_ENVIRONMENTS: 1       # number of environments (per agent process)
 DUMP_LOCATION: datadump   # path to dump models and log
 EXP_NAME: eval_floorplanner       # experiment name
 VISUALIZE: 0              # 1: render observation and predicted semantic map, 0: no visualization
 PRINT_IMAGES: 1           # 1: save visualization as images, 0: no image saving
-GROUND_TRUTH_SEMANTICS: 0 # 1: use ground-truth semantics (for debugging / ablations)
-seed: 0
+GROUND_TRUTH_SEMANTICS: 1 # 1: use ground-truth semantics (for debugging / ablations)
```

Comment:
Currently this will skip running detic in ovmm agent. Chris has added a check for this in the config loader for the stretch env, so it'll fail early with an understandable error message.